README.md

```diff
@@ -1,12 +1,22 @@
-## Opportunity Calendar
+## Opportunity Calendar <img src="https://github.com/Samridhi-98/Images/blob/master/Images/op-logo.png" width="3%">
 
+<p>
+
+[![GitHub contributors](https://img.shields.io/github/contributors/adityabisoi/REST-fetch)](https://github.com/Girl-Code-It/Opportunity-Calendar-Backend/graphs/contributors/)
+[![Issues](https://img.shields.io/github/issues/Girl-Code-It/Opportunity-Calendar-Backend)](https://github.com/Girl-Code-It/Opportunity-Calendar-Backend/issues)
+[![PRs](https://img.shields.io/github/issues-pr/Girl-Code-It/Opportunity-Calendar-Backend)](https://github.com/Girl-Code-It/Opportunity-Calendar-Backend/pulls)
+[![Forks](https://img.shields.io/github/forks/Girl-Code-It/Opportunity-Calendar-Backend)](https://github.com/Girl-Code-It/Opportunity-Calendar-Backend)
+[![Stars](https://img.shields.io/github/stars/Girl-Code-It/Opportunity-Calendar-Backend)](https://github.com/Girl-Code-It/Opportunity-Calendar-Backend)
+[![Visits Badge](https://badges.pufler.dev/visits/Girl-Code-It/Opportunity-Calendar-Backend)](https://badges.pufler.dev)
```

Comment:
We don't need this one as of now

Reply:
Is it wrong? 
Do we have to add shield badges naah?